### ISSUE TITLE
Aspect for two organizations

### ISSUE THREAD

**Issue by fnielsen** [fnielsen/scholia#107]
Aspect for two organizations

Panel for instance, a co-worker list:

```
SELECT ?author1 ?author1Label ?author2 ?author2Label
WITH {
  SELECT DISTINCT ?author1 ?author2 WHERE {
    { ?author1 wdt:P108 wd:Q1269766 . } union { ?author1 wdt:P1416/wdt:P361* wd:Q1269766 . }
    { ?author2 wdt:P108 wd:Q4614 . } union { ?author2 wdt:P1416/wdt:P361* wd:Q4614 . }
    ?work wdt:P50 ?author1 .
    ?work wdt:P50 ?author2 .
  } 
} AS %result
WHERE {
  INCLUDE %result 
  SERVICE wikibase:label { bd:serviceParam wikibase:language "en,da,de,es,fr,jp,sv,ru,zh," . }
}
```

**Comment by fnielsen** [comment]
Part of #71

**Comment by fnielsen** [comment]
This has been implemented with cb9d8f655bf47124872f2831ab8ed20c9e84f6ff and is in production, see, e.g., https://tools.wmflabs.org/scholia/organizations/Q1269766,Q193196
